providers/src/airflow/providers/databricks/operators/databricks.py

```diff
@@ -711,12 +717,15 @@ class DatabricksRunNowOperator(BaseOperator):
     Currently the named parameters that ``DatabricksRunNowOperator`` supports are
         - ``job_id``
         - ``job_name``
+        - ``job_parameters``
         - ``json``
+        - ``dbt_commands``
         - ``notebook_params``
         - ``python_params``
         - ``python_named_parameters``
         - ``jar_params``
         - ``spark_submit_params``
+        - ``sql_params``
```

Comment:
We shouldn't add support for `sql_params` at this point. They are [deprecated in the API](https://docs.databricks.com/api/workspace/jobs/runnow#sql_params), and passing `job_parameters` to the job run will automatically push them to every SQL task in the job.

Reply:
Sure.